mirrord/config/src/fs/advanced.rs

```diff
@@ -14,29 +14,36 @@ use crate::{
 ///
 /// Allows the user to specify:
 ///
-/// - `MIRRORD_FILE_OPS` and `MIRRORD_FILE_RO_OPS`;
-/// - `MIRRORD_FILE_FILTER_INCLUDE` and `MIRRORD_FILE_FILTER_EXCLUDE`;
+/// What is the default behavior for file operations:
+/// 1. read - Read from the remote file system (default)
+/// 2. write - Read/Write from the remote file system.
+/// 3. local - Read from the local file system.
+/// 4. disabled - Disable file operations.
 ///
-/// ## Examples
-///
-/// - Read-only excluding `.foo` files:
+/// Besides the default behavior, user can specify behavior for specific regex patterns. Case
+/// insensitive. 1. read_write - List of patterns that should be read/write remotely.
+/// 2. read_only - List of patterns that should be read only remotely.
+/// 3. local_only - List of patterns that should be read locally.
 ///
-/// ```yaml
-/// # mirrord-config.yaml
+/// The logic for choosing the behavior is as follows:
+/// 1. Check if one of the patterns match the file path, do the corresponding action. There's no
+/// specified order    if two lists match the same path, we will use the first one (and we do not
```

Comment:
Would write that "Specifying the same path in two lists is unsupported and can lead to undefined behaviour." Would also log a warning when it happens.

Reply:
We can't know that unless we process all Regexes then see if more than one match which feels like an overkill.

Reply:
Okay but the log comment still stands